packages/expo-asset/plugin/src/utils.ts

```diff
@@ -0,0 +1,33 @@
+import fs from 'fs/promises';
+import path from 'path';
+
+export const imageTypes = ['.png', '.jpg', '.gif'];
```

Comment:
What about webp or avif?

Reply:
Neither are supported in Xcodes asset catalog. They need special handling.